pkg/provider/onelogin/onelogin.go

```diff
@@ -360,7 +360,7 @@ func verifyMFA(oc *Client, oauthToken, appID, resp string) (string, error) {
 			switch gjson.Get(string(body), "status.type").String() {
 			case TypePending:
 				time.Sleep(time.Second)
-				fmt.Print(".")
+				log.Print(".")
```

Comment:
Not that useful to log?

Reply:
I've removed this and added a `logger.Debug` so now both `okta.go` and `onelogin.go` don't print out the individual dots but if you turn on debug you will see the output while waiting.